docs/multi-organization-guide.md

```diff
@@ -0,0 +1,182 @@
+3. Allow for empty deployment map targets
+    - With the adfconfig setting `allow-empty-target` ([ADF Admin Guide](admin-guide.md)), ADF can be instructed to ignore any target that is not resolvable or empty (because no AWS Accounts exists in it). It is suggested to set this setting to `True`. Eventhough the OU structure and general setup across the different AWS Organization stages is usually identical, the number of created AWS Accounts might not. With this setting to `True`, temporary empty OUs are just ignored and do not lead to an error.
+
+ 4. The source branch for the application code may be different per AWS Organization
+    - The above described custom `adfconfig` configuration allows a different default 
+  branch to be specified in the path `config.scm.default-scm-branch` per AWS Organization
+
+### 3. Make the AWS Organization Stage Context Available in Codepipeline Build jobs
+ADF applications often contain environment / AWS Organization stage specific configuration files. 
+In order to allow AWS Codebuild to select the proper configuration context for an application, the environment / AWS Organization stage context needs to be made available. 
+A simple pattern to solve this problem is the introduction of the SSM parameter `adf/org/stage` in  the buildspec file of the application.
+The following snippet shows the header of such a `codebuild.yaml` file. 
+```
+env:
+  parameter-store:
+    ADF_ORG_STAGE: "/adf/org/stage"
+[...]
+```
+This environment variable can then be used to drive decision/deployment logic
+within any of the subsequent build commands/actions.
+Some scenarios which could require Org specific context:
+- Deriving the default log level based on the org stage for
+a specific CDK application 
+- Appending the Stage name to AWS resource names having a requirement to be 
+both deterministic as well as globally unique 
+(whilst being deployed into multiple organizatinos)
+- Selection a config file from a config folder with the following files:
+    - `config-dev.yaml`
+    - `config-int.yaml`
+    - `config-prod.yaml`
+
+### 4. Customize the Base IAM Roles Per Organization
+ADF Supports Bootstrapping Baseline Cloudformation Stacks to all accounts
+when they first join an AWS Organization and centrally governing the subsequent 
+Lifecycle of those Stacks. [See Here](admin-guide.md#bootstrapping-accounts) 
+
+These Baseline Templates are typically used for Setting up Default IAM Roles and
+Policies necessary for the foundations of an ADF Based Enteprise Landing Zone. 
+
+In guidance with AWS Security Guidelines and `Least Privilege Access Principles`,
+it it recommended to reduce the scope of any IAM Policy to the minimum required
+Actions, Principals and Resource Scope necessary. 
+
+To customize the scope of which resources or Principals are permitted within the
+IAM Policies of the Baseline templates CFN Mapping fields can be utilized based 
+on the `Org Stage` SSM Parameter. As shown below:
+
+```
```

Comment:
Can you add the `yaml` keyword directly after these three backticks?

Reply:
done